### PR TITLE
Creation of the class bullet for space invators shooter

### PR DESCRIPTION
**Pull request by danielaSantaCruz**
Hello team here is the class bullet for review

### CODE REVIEW

**Comment by J0c4** [comment]
Nice baby steps guys, now lets try to add unit tests for this code and this PR should be good to go!

File: src/main/java/org/fundacionjala/at13/spaceinvaders/bullet.java [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    public void shooter(String type){
+        if(bulletExist){
+            System.out.println("The " + type + " has fired from " + this.positionInitialX + " " + this.positionInitialY);
+            bulletExist = false;
+        }else{
+            System.out.println("the " + type + " cannot fired");
+        }
+    }
+    public boolean isCollapsedBullet(int x1, int y1, int x2, int y2){
```
Comment (cristianfchq) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/bullet.java:21]: maybe the names of the method parameters should be more descriptive.

Comment (jpgonzales27) [inline src/main/java/org/fundacionjala/at13/spaceinvaders/bullet.java:21]: in the bullet method be careful with the spacing of the parameters

**Review by danielaSantaCruz — COMMENTED** [review]
Add the unit test for methods: 
* shooter
* isCollapsedBullet
Solved the review comments

**Pull request merged**